### ISSUE TITLE
get.trait.data deletes too many files

### ISSUE THREAD

**Issue by ryankelly-uiuc** [PecanProject/pecan#581]
Or at least, I have a problem because my PDA code writes files to the PFT dirs, but I am then interested in rerunning parts of the workflow to add new analyses. If I re-run `get.trait.data()` , my previous results are lost. 

I guess this makes some sense, because if `get.trait.data()` updates the priors, then any analyses that were done with the previous priors are now inconsistent with the new ones. But it's easy to inadvertently re-run `get.trait.data()` when going back to the workflow, and accidentally blast useful output. I think I have the provenance stuff working well enough now that all of those results should be reproducible anyway, but still thought it might be worth discussing as an issue.

And most of all, I wanted to draw attention to this so that 2015 Flux Campers can *warn their students*. It seems likely that they will be running multiple rounds of PDA mcmc. If done over multiple R sessions, this requires `read.settings()` (which occurs before `get.trait.data()` in the workflow) and then one or more tasks (e.g. `run.write.configs()`, EA, SA, and/or PDA) that all occur *after* `get.trait.data()`. Very easy to screw up!

**Comment by mdietze** [comment]
Understood, but if the meta-analysis is set to AUTO and you haven't added any trait data it shouldn't do anything even if you re-run get.trait.data. If it is that seems like a bug that should be fixed.  

That said, it shouldn't delete the PDA output regardless, even if the MA does need to be rerun, so that's a second bug.

**Comment by ryankelly-uiuc** [comment]
>Understood, but if the meta-analysis is set to AUTO and you haven't added any trait data it shouldn't do anything even if you re-run get.trait.data. If it is that seems like a bug that should be fixed.

No, should have clarified that this was with MA=TRUE for testing purposes. 

> That said, it shouldn't delete the PDA output regardless, even if the MA does need to be rerun, so that's a second bug. 

OK, then I would call this a bug. I either don't know how or am unable to label it as one, though.
